doctrinal-works-v1/original/37-chapter-35.adoc

```diff
@@ -148,4 +159,5 @@ in which spirit and truth, the God of all truth, who is a spirit, is worshipped.
 From a lover of your souls, and eternal good,
 and that righteousness and truth may be established among you.
 
+[.signed-section-signature]
 G+++.+++ F. a prisoner in Lancaster Castle.
```

Comment:
Does this look okay like this? I thought it might have looked a little funny. 😄

Reply:
I moved the `a prisoner in Lancaster Castle` down to be a `[.signed-section-context-close]` -- seemed a little better that way.